docs/docs/reference/release-notes/v2.30/index.md

```diff
@@ -0,0 +1,189 @@
+
+And several impactful updates in the new [`gatsby-plugin-image`](#gatsby-plugin-image050-beta) (beta):
+
+- [AVIF support](#avif-support) - enjoy next-gen image format
+- [Support remote static images](#support-static-image-urls-experimental) - download, transform and optimize remote images with a single line
+- [New defaults](#new-defaults) - less configuration for most common image setups
+
+Also check out [notable bugfixes](#notable-bugfixes).
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](/docs/reference/release-notes/v2.29)
+
+[Full changelog](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.30.0-next.0...gatsby@2.30.0)
+
+## Query on Demand and Lazy Images: Generally Available
+
+Those two features change the way how `gatsby develop` works:
+
+- We no longer transform all images eagerly on dev server start
+- We no longer run all queries
+
+Instead, it happens lazily when the browser requests a page or a specific image variant.
+This significantly shortens the boot-up time for `gatsby develop`.
+
+> Note that it does not affect `gatsby build` in any way, only `gatsby develop`
+
+We've been gradually rolling out those features:
+
+- "Query on Demand" was introduced behind the flag in [v2.27](/docs/reference/release-notes/v2.27#experimental-queries-on-demand).
+- "Lazy images" were introduced behind the flag in [gatsby-plugin-sharp@2.10.0](/docs/reference/release-notes/v2.28#experimental-lazy-images-in-develop).
+- In [v2.29](/docs/reference/release-notes/v2.29#queries-on-demand) 10% of our users were automatically opted-in
+
+This allowed us to collect the feedback and fix bugs as they were discovered.
+
+Now we feel confident enough to publish it as generally available. You don't need to add flags or any environment
+variables to enable it. Just upgrade to `gatsby@2.30.0` (and `gatsby-plugin-sharp@2.12.0` for lazy images).
+
+In v2.29 we improved the UX around long-running queries by adding a loading indicator and message to the browser console (only in `gatsby develop`). If you want or need to de-activate this indicator, you can! For more details please see the [umbrella discussion](https://github.com/gatsbyjs/gatsby/discussions/27620).
+
+## Server Side Rendering (SSR) in development
+
+There are certain types of build errors that currently are only discoverable when doing a build. The most common is code that tries to access browser globals (like `window`) that don't exist in Node.js when SSRing your Gatsby site.
+
+This causes a lot of frustration as you might develop for days before running a build and it’s not fun to discover problems long after they were created. Then actually fixing the problems is painful as the feedback cycle is slow as you have to run a build after each code change.
```

Comment:
Do we want to mention that in dev-ssr mode `style-loader` is no longer used and that styling inconsistencies between develop and build _might_ be addressed? (I think there are some edge cases when it's not exactly the same due to us forcing single css chunk for prod, but for dev it's possibly to create more when doing lazy imports?)

/cc @KyleAMathews